storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/integration/IntegrationTest.java

```diff
@@ -307,4 +312,32 @@ public void call(Changes changes) {
 
         return storedTweets;
     }
+
+    @NonNull
+    PreparedGetCursor createQueryWithNullResult() {
```

Comment:
We need test `DefaultStorIOContentResolver` with it's not mocked `query` method. But we can't instantiate it, because in constructor it's call looper